Resources/Private/Language/de.locallang.xlf

```diff
@@ -495,6 +797,26 @@ Sächsische Landesbibliothek- Staats- und Universitätsbibliothek Dresden
 			<source>You have registered a document</source>
 			<target>Ihre Veröffentlichung auf Qucosa – Anmeldebestätigung</target>
 		</trans-unit>
+		<trans-unit id="notification.registerDocument.admin.body" approved="yes" xml:space="preserve">
```

Comment:
The email message are usually configured in the client record via TYPO3 backend. Is this no longer the case here? If so, why?

Reply:
But default messages have always been configured, this is done in the the same way as before, so it is not mandatory to configure them in the client 